src/__tests__/resources/3d.mock.spec.ts

```diff
@@ -36,11 +45,47 @@ describe('3D mocked', async () => {
       },
     },
   ];
+  const revisionReveal: RevealRevision3D = {
+    ...revision,
+    sceneThreedFiles: [
+      {
+        version: randomInt(),
+      },
+    ],
+  };
+  const nodesReveal: RevealNode3D[] = nodes.map(node3D => ({
+    ...node3D,
+    sectorId: randomInt(),
+  }));
+  const sectors: RevealSector3D[] = [
+    {
+      id: randomInt(),
+      path: 'ftps/stairway/to/heaven',
```

Comment:
This will always be number based `0/3/7/1`

Reply:
oh, ok ;(